api/filter_dimensions_test.go

```diff
@@ -801,3 +803,159 @@ func TestCreatePublicDimensionsSucceeds(t *testing.T) {
 		So(publicDims[1].Links.Options.HRef, ShouldEqual, "/filters/5678/dimensions/testDim2/options")
 	})
 }
+
+func TestPutFilterBlueprintDimension(t *testing.T) {
+	stubBody := `{"name":"Test dimension","dimension_url":"http://localhost:22000/datasets/cantabular-flexible-example/editions/2021/versions/1/dimensions/city/options","is_area_type":true}`
```

Comment:
This might look a little neater spread over multiple lines. You can have new lines in string literals using back ticks so shouldn't be any hassle